src/librustc/hir/lowering.rs

```diff
@@ -1602,7 +1663,12 @@ pub fn lower_expr(lctx: &LoweringContext, e: &Expr) -> P<hir::Expr> {
                     // `{ let _result = ...; _result }`
                     // underscore prevents an unused_variables lint if the head diverges
                     let result_ident = lctx.str_to_ident("_result");
-                    let let_stmt = stmt_let(lctx, e.span, false, result_ident, match_expr, None);
+                    let let_stmt = stmt_let(lctx,
+                                            e.span,
+                                            false,
+                                            result_ident,
+                                            match_expr,
+                                            None);
```

Comment:
nit: this doesn't need to be changed (unless you prefer the new formatting -- I usually prefer to save vertical space)